### ISSUE TITLE
sdsl/construct_sa.hpp:28:10: fatal error: divsufsort.h: No such file or directory

### ISSUE THREAD

**Issue by shenwei356** [U-Ar/br-index#4]
Hi, I tried to install this package many times but still failed.

[sdsl-lite](https://github.com/simongog/sdsl-lite) is installed as in `~/sdsl`, so there's no need to edit `CMakeLists.txt`. However it still report that `divsufsort.h` was not found, which is actually in `external/libdivsufsort/include/`.

**Comment by U-Ar** [comment]
Maybe `divsufsort.h` is not included.
Do you have `divsufsort.h` in `~/sdsl/include` ?
If you install sdsl-lite according to the instructions, it should be placed in that location.
Otherwise, you will need to explicitly include the directory where `divsufsort.h` is located.

**Comment by shenwei356** [comment]
I see, I figured it out. When installing sdsl-lite, I ran the command below.

    ./install.sh .

So the include and library files will be located in the sdsl-lite project itself.

However, according to the command from sdsl-lite

    ./install.sh

The include and library files will be located in $HOME. Therefore, br-index could not locate `divsufsort.h`.

It's recommended to add a note in [System Requirements](https://github.com/U-Ar/br-index#system-requirements).

Thanks for your prompt reply.
